src/loaders/common/loader_common_names.py

```diff
@@ -24,13 +26,31 @@
 File structure at NERSC for loader programs
 """
 
-ROOT_DIR = '/global/cfs/cdirs/kbase/collections'  # root directory for the collections project
-SOURCE_DATA_DIR = 'sourcedata'  # subdirectory for all source data (e.g. GTDB genome files)
-COLLECTION_DATA_DIR = 'collectionsdata'  # subdirectory for collected data (e.g. computed genome attributes)
+ROOT_DIR = (
+    "/global/cfs/cdirs/kbase/collections"  # root directory for the collections project
+)
+SOURCE_DATA_DIR = (
+    "sourcedata"  # subdirectory for all source data (e.g. GTDB genome files)
+)
+COLLECTION_DATA_DIR = "collectionsdata"  # subdirectory for collected data (e.g. computed genome attributes)
 
 # metadata file generated in the tool result folder with tool generated genome identifier,
 # original genome id and source genome file info
-GENOME_METADATA_FILE = 'genome_metadata.tsv'
+GENOME_METADATA_FILE = "genome_metadata.tsv"
 
 # the name of the CSV file where we store the trait counts
-TRAIT_COUNTS_FILE = 'trait_counts.csv'
+TRAIT_COUNTS_FILE = "trait_counts.csv"
+
+
+# kbase authentication token
+KB_AUTH_TOKEN = "KB_AUTH_TOKEN"
+# subdirectory for SDK jobs per user
+SDK_JOB_DIR = "sdk_job_dir"
+# used by the podman service. Note that the unix user ID must be interpolated into the string before use
+DOCKER_HOST = "unix:/run/user/{}/podman/podman.sock"
+# used by the callback server
+KB_BASE_URL_DEFAULT = "https://kbase.us/services/"
+# JSON keys in the download metadata file in a download directory
+SOURCE_METADATA_FILE_KEYS = ["upa", "name", "type", "timestamp"]
+# callback server docker image name
+CALLBACK_IMAGE_NAME = "scanon/callback"
```

Comment:
Probably ok for now, but we might need to add a tag so we always pull the same version of the server. Something to keep in mind for the future